design-documents/graph-ql/coverage/customer-orders.md

```diff
@@ -325,6 +325,30 @@ type CommentItem {
 }
 ```
 
+## OrderAddress type
+```graphql
+
+type OrderAddress @doc(description: "OrderAddress contains detailed information about the order billing and shipping addresses"){
+    firstname: String @doc(description: "The first name of the person associated with the shipping/billing address")
+    lastname: String @doc(description: "The family name of the person associated with the shipping/billing address")
+    middlename: String @doc(description: "The middle name of the person associated with the shipping/billing address")
+    region: String @doc(description: "The state or province name")
+    region_id: Int @doc(description: "The unique ID for a pre-defined region")
```

Comment:
Seems like `region_id` probably isn't needed here (any region data should be available from the `OrderAddress.region` field

Reply:
Whoops - I thought `OrderAddress.region` was assigned an Object Type, but it's just a `String`.

Instead, let's switch `region_id` field to an `ID` type. The long-term plan is for unique identifiers to be backed by the `ID` type, and this will make `OrderAddress.region_id` ready for that change.

[More Context](https://github.com/magento/architecture/pull/396)

Reply:
Sounds good to me, I will change the 'OrderAddress.region_id' to type 'ID'